Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -875,10 +877,11 @@ TccModePostMemConfig (
         FspsUpd->FspsConfig.PcieRpAspm[Index]        = PolicyConfig->PchPcieAspm;
         FspsUpd->FspsConfig.PcieRpL1Substates[Index] = PolicyConfig->PchPcieRpL1;
       }
-
-      mTccRtd3Support    = PolicyConfig->Dstates;
-      mTccLowPowerS0Idle = PolicyConfig->Sstates;
       mTccDsoTuning      = TRUE;
+      if (mPchSciSupported != 0) {
+        mTccRtd3Support                = PolicyConfig->Dstates;
+        FeaturesCfgData->Features.S0ix = PolicyConfig->Sstates;

```

Comment:
Had better not update SBL cfg data. If needed you could use a local variable.

Reply:
You are right. I have introduced S0ix flag in PlatformData.h. Hope that works.